source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -904,7 +904,7 @@ message TransformationEquationInstruction {
   // equation is guaranteed to hold at any program point where all ids involved
   // are available (i.e. at any program point dominated by the instruction).
 
-  // The result id of the new instruction
+  // Fresh ids required by this transformation.
```

Comment:
I think you should revert the comment, since there is only one fresh id once more.

Reply:
Fixed in a6ad6b3.